### PR TITLE
create ruby-version through rbenv

### PR DESCRIPTION
**Pull request by d4be4st**
Setting .ruby-version

### CODE REVIEW

**Comment by melcha** [comment]
I agree with @DamirSvrtan, I think we should somehow enforce the newest Ruby version. 

I suggest asking a question at the top of the template, something like this:
```
if yes?(“Rbenv global returned 2.4.2. Is this the latest stable Ruby version?")
  # do everything
else
    puts “Please install the latest Ruby version and set it as global."
end
```

It would be great if we could fetch the latest Ruby version via some API and compare it to the rbenv global output, but this will also work.

**Comment by d4be4st** [comment]
Ok, new take @infinum/backend-team !

Template now tries to update ruby-build. As i know half of us have it installed through homebrew and other half as a plugin i have taken precautions.

After ruby-build update, rbenv finds latest stable ruby build and installs it.

File: template.rb [hunk 1]
```diff
@@ -259,22 +259,44 @@
 append_file '.gitignore', GITIGNORED_FILES
 
 # Finish
+
+## Install latest ruby and sets it as local version
+
+say('Updating ruby-build and installing latest ruby', :cyan)
+
+if !run('brew info ruby-build', capture: true).include?('Not installed')
```
Comment (DamirSvrtan) [inline template.rb]: Everything is fine I'd just reverse the order of if else so you don't have to use negations in the first clause.

**Comment by DamirSvrtan** [comment]
@infinum/backend-team if there aren't any comments I'd advise @d4be4st to merge it in.

**Pull request merged**